draft-looker-oauth-attested-key-based-client-authentication.md

```diff
@@ -94,34 +94,51 @@ The following diagram depicts the conceptual interactions.
                      +------->|               |<------------------------>|               |
   (1) Generate       |        |    Client     |    (6) Interaction       | Authorization |
   Attestation Key    |        |   Instance    |    using Client          |    Server     |
-                     +--------|               |    Key Attestation + PoP |               |
+                     +--------|               |    Attestation + PoP     |               |
                               |               |    for authentication    |               |
                               +---------------+                          +---------------+
                                  / \      |
                                   |       |
                                   +-------+
                                 (5) Generate
-                              Key Attestation PoP
+                           Client Attestation PoP
 ~~~
 
-Note defining the protocol for steps 2 and 4, including how the client instance authenticates with the client backend is out of scope of this specification.
+At first, the Authorization Server will communicate its requirements for the client authentication through its metadata. A Client instance that wants to request an access token from this Authorization Server will need to obtain a Client Attestation JWT, for example, from its client backend first.
```

Comment:
I'd write using present, removing "will"

WDYT about

`The flow starts with the AS communicating its requirements for the client authentication to the client, made available in its metadata.`

Having said "the AS" that would be the same for the Client as well, thus

`The Client instance makes available its Client Attestation JWS to the AS where an access token is requested`